e2e/cypress/integration/integrations/incoming_webhook/long_url_embedded_image_spec.js

```diff
@@ -0,0 +1,93 @@
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+describe('Integrations', () => {
+    let testUser;
+    let testTeam;
+    let testChannel;
+    let newIncomingHook;
+    let incomingWebhook;
+
+    before(() => {
+        // # Create new setup
+        cy.apiInitSetup().then(({user}) => {
+            testUser = user;
+
+            // # Login as the new user
+            cy.apiLogin(testUser).then(() => {
+                // # Create a new team with the new user
+                cy.apiCreateTeam('test-team', 'Team Testers').then(({team}) => {
+                    testTeam = team;
+
+                    // # Create a new test channel for the team
+                    cy.apiCreateChannel(testTeam.id, 'test-channel', 'Testers Channel').then(({channel}) => {
+                        testChannel = channel;
+
+                        // # Declare web-hook values
+                        newIncomingHook = {
+                            channel_id: testChannel.id,
+                            channel_locked: true,
+                            description: 'Test Webhook Description',
+                            display_name: 'Test Webhook Name',
+                        };
+
+                        //# Create a new webhook
+                        cy.apiCreateWebhook(newIncomingHook).then((hook) => {
+                            incomingWebhook = hook;
+                        });
+
+                        // # Visit the test channel
+                        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+                    });
+                });
+            });
+        });
+    });
+
+    it('MM-T643 Incoming webhook:Long URL for embedded image', () => {
+        const letters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+        const queries = letters.split('').reduce((acc, letter) => {
+            const newValue = acc + `&${letter}=${letters}`;
+            return newValue;
+        }, '');
+        const url = `http://via.placeholder.com/300.png?expires=213134234234234234234234${queries}`;
+        const payload = getPayload(testChannel, url);
+
+        // # Post the webhook message
+        cy.postIncomingWebhook({url: incomingWebhook.url, data: payload});
+
+        // * Assert that the message was posted
+        cy.uiWaitUntilMessagePostedIncludes('Hey attachments');
+        cy.getLastPostId().then(() => {
+            const baseUrl = Cypress.config('baseUrl');
+            const encodedUrl = `${baseUrl}/api/v4/image?url=${encodeURIComponent(url)}`;
+
+            // * Assert that file image is present
+            cy.findByLabelText('file thumbnail').should('be.visible').and('have.attr', 'src', encodedUrl);
+
```

Comment:
Thanks Mike for updating,
can you replace lines from 77 to 81 by following lines,
```
cy.findByText('Show more').should('be.visible').click();
cy.findByText('Show less').scrollIntoView().should('be.visible');
```
FindByText is basically finding element by css text property so we don't need to add ('have.text') assertion